backend/compact-connect/tests/app/test_api/test_provider_management_api.py

```
@@ -20,6 +20,163 @@ class TestProviderManagementApi(TestApi):
     4. Ensure the request and response models for the endpoint are present and match the expected schemas.
     """
 
+    def _get_privilege_encumbrance_resource_id(self, api_stack_template, api_stack):
+        """Helper method to get the privilege encumbrance resource ID by traversing the resource hierarchy."""
```

Pylint (3.3.7)

[convention] Line too long (110/100)

(C0301)